src/transformers/modeling_tf_utils.py

```diff
@@ -2859,7 +2922,16 @@ def from_pretrained(
         if filename == SAFE_WEIGHTS_NAME:
             with safe_open(resolved_archive_file, framework="tf") as f:
                 safetensors_metadata = f.metadata()
-            if safetensors_metadata is None or safetensors_metadata.get("format") not in ["pt", "tf", "flax"]:
+            if safetensors_metadata is None or safetensors_metadata.get("format") not in ["pt", "tf", "flax", "mlx"]:
```

Comment:
Looks like we might want a constant e.g. `SUPPORTED_SAFE_FORMATS = ["pt", "tf", "flax", "mlx"]` so we don't have to update this in several locations here and for PT (for another PR)

Reply:
Agreed!